packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -437,6 +439,10 @@ class Padding extends SingleChildRenderObjectWidget {
 /// unconstrained, then in that direction it will be sized to fit the child's
 /// dimensions. Using widthFactor and heightFactor you can force this latter
 /// behavior in all cases.
+///
+/// See also:
+///
+///  * [CustomSingleChildLayout]
```

Comment:
` * [Center] (which is the same as Align but with the [alignment] always set to 0.5,0.5)` or some such